website/integrations/services/atlassian/index.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: Integrate with Atlassian Cloud
+sidebar_label: Atlassian Cloud
+support_level: community
+---
+
+## What is Atlassian Cloud
+
+> Atlassian is a proprietary software company that specializes in collaboration tools designed primarily for software development and project management. Atlassian Cloud is their cloud platform and provides access to their popular apps; Jira, Confluence, Bitbucket, Trello and others.
+>
+> -- https://www.atlassian.com/
+
+:::important
+This guide offers instructions for setting up authentik as a SAML provider specifically for Atlassian Cloud. It is applicable to all Atlassian Cloud applications, including Jira, Confluence, Bitbucket, Trello, and others.
+
+SAML SSO for Atlassian Cloud apps requires an [Atlassian Guard](https://support.atlassian.com/security-and-access-policies/docs/understand-atlassian-guard/) subscription and a [verified domain](https://support.atlassian.com/user-management/docs/verify-a-domain-to-manage-accounts/). Further information on requirements for SSO can be found in the [Atlassian SSO documentation](https://support.atlassian.com/security-and-access-policies/docs/configure-saml-single-sign-on-with-an-identity-provider/).
```

Comment:
Consider moving this paragraph about Atlassian Guard being required down to the Preparation section, because it is a pre-req. It gets a little lost within this big Note.